api/src/main/java/com/strategyobject/substrateclient/api/Api.java

```diff
@@ -1,7 +1,33 @@
 package com.strategyobject.substrateclient.api;
 
+import com.strategyobject.substrateclient.pallet.GeneratedPalletResolver;
 import com.strategyobject.substrateclient.rpc.Rpc;
+import com.strategyobject.substrateclient.rpc.RpcImpl;
+import com.strategyobject.substrateclient.transport.ProviderInterface;
+import lombok.val;
 
+/**
+ * Provides the ability to query a node and interact with the Polkadot or Substrate chains.
+ * It allows interacting with blockchain in different ways: using RPC's queries directly or
```

Comment:
It allows interacting with blockchain in two different ways: using RPC's queries directly or
 OR
 It allows interacting with blockchain in various ways: using RPC's queries directly or